dubbo-common/src/main/java/org/apache/dubbo/common/utils/MethodUtils.java

```diff
@@ -21,6 +21,21 @@
 
 public class MethodUtils {
 
+    /**
+     * return true if and only if:
+     * <p>
+     * 1, public
+     * <p>
+     * 2, name starts with "set"
+     * <p>
+     * 3, only has one parameter
+     */
+    public static boolean isSetter2(Method method) {
```

Comment:
it is a bad idea to have two *isSetter* methods, pls. consider to combine both into one single isSetter method

Reply:
> it is a bad idea to have two _isSetter_ methods, pls. consider to combine both into one single isSetter method

this is an isSetter method in class, this two are different , let me try to diff them